lib/percpu_counter.c

```diff
@@ -81,7 +81,7 @@ void __percpu_counter_add(struct percpu_counter *fbc, s64 amount, s32 batch)
 	if (count >= batch || count <= -batch) {
 		unsigned long flags;
 		raw_spin_lock_irqsave(&fbc->lock, flags);
-		fbc->count += count;
+		ACCESS_ONCE(fbc->count) += count;
```

Comment:
please use newer WRITE_ONCE, it will be:
WRITE_ONCE(fbc->count, READ_ONCE(fbc->count) + count);